docs/dev_101_services.md

```diff
@@ -197,3 +197,62 @@ If you need more control over the service call, you can also pass an async funct
 async def custom_set_sleep_timer(entity, service_call):
     await entity.set_sleep_timer(service_call.data['sleep_time'])
 ```
+
+## Response Data
+
+Services may optionally respond to a service call with data for powering more advanced automations. The use of response data for services is meant for
+cases that do not fit the Home Assistant state. For example, a response stream of repeated objects, or API would be better queried on demand or filtered such as a search.
```

Comment:
This could use some more concrete usage examples. What I was wondering is whether it would be fine to use this for "raw  command" service calls to allow users act on devices in ways that are not yet-if-ever supported by the integration?